usaspending_api/settings.py

```diff
@@ -6,13 +6,17 @@
 import os
 from pathlib import Path
 
-import ddtrace
 import dj_database_url
 from django.db import DEFAULT_DB_ALIAS
 from django.utils.crypto import get_random_string
 
 from usaspending_api.config import CONFIG
 
+TRACE_ENV = "local"
```

Comment:
Follow a similar pattern to the `ENV_CODE` variable

Reply:
Removed ENV_CODE from settings.py as it is unnecessary here.